infra/cifuzz/fuzz_target.py

```diff
@@ -121,6 +124,34 @@ def is_reproducible(self, test_case):
         return True
     return False
 
+  def is_crash_valid(self, test_case):
+    """Checks if a crash is reproducible, and if it is, whether it's a new
```

Comment:
I think the terms "novel" (consequential is a bit more descriptive but I usually see this word to mean important so might be confusing) for bugs that are introduced in a PR and "not_novel" for bugs reproing on OSS-Fuzz are a bit nicer than exists in old